### DISCUSSION TITLE
Many thanks to all who make this happen!

### DISCUSSION THREAD

**Discussion by plasmac0r** [jonaswinkler/paperless-ng#1025] [General]
Hello Jonas and all who contributed!

I'm using paperless-ng for some weeks now, put all my documents in it and have them well sorted now. 
It's easy and fun to use! The documentation helped me a lot and I think it's pretty good too!

I would like to thank you for your work a 1000 times!

Regards
plasmac0r

**Comment by knilde** [comment]
Yeah, thank you for great work! 😃
